packages/terra-form-select/tests/jest/SearchableFields.test.jsx

```diff
@@ -8,7 +8,7 @@ import TagSelectField from '../../src/TagSelectField';
 
 it('should render the selectId to the input field in ComboboxField component', () => {
   const select = (
-    <ComboboxField label="Label" defaultValue="blue" selectId="select-id">
+    <ComboboxField label="Label" defaultValue="blue" selectId="select-id" maxHeight={100}>
```

Comment:
Please create new snapshot tests for these that indicate the desired test. Adding maxHeight to existing tests makes future updates more confusing because the test description does not indicate this should test max height.

> should render the selectId to the input field in ComboboxField component